app/src/main/java/org/fossasia/phimpme/sharetwitter/HelperMethods.java

```diff
@@ -50,22 +47,40 @@ public static void postToTwitterWithImage(Context context, final String imageUrl
 		final Twitter twitter = new TwitterFactory(configuration).getInstance();
 
 		final File file = new File(imageUrl);
-		boolean success = true;
-		if (file.exists()) {
-			try {
-			StatusUpdate status = new StatusUpdate(message);
-			status.setMedia(file);
-				twitter.updateStatus(status);
-			} catch (TwitterException e) {
-				e.printStackTrace();
-				success = false;
-			}
-		}else{
-			Log.d(TAG, "----- Invalid File ----------");
-			success = false;
-		}
-		postResponse.onFinsihed(success);
 
+		new Thread(new Runnable() {
```

Comment:
You are creating a new thread it is already running in the background thread. Aysync task provides a background thread so no need to create a new thread here.